device-registry/src/main/scala/org/genivi/sota/device_registry/SystemInfo.scala

```diff
@@ -39,6 +42,19 @@ object SystemInfo extends SlickJsonHelper {
 
   val systemInfos = TableQuery[SystemInfoTable]
 
+  private def addUniqueIdsSIM(j: Json): State[Int, Json] = j.arrayOrObject(
+    State.pure(j),
+    _.traverseU(addUniqueIdsSIM).map(Json.fromValues),
+    _.toList.traverseU {
+      case ("id", value) => State { (nr: Int) =>
+        (nr+1, Seq("id" -> value, "uuid" -> Json.fromString(s"$nr")))
```

Comment:
uuid will be just a number, so we shouldn't call it uuid as this name has some implicit semantics (https://en.wikipedia.org/wiki/Universally_unique_identifier)